content/500-platform/10-about.mdx

```diff
@@ -5,13 +5,133 @@ metaDescription: 'Learn about the Platform Console to integrate the Prisma Data
 displayed_sidebar: platformSidebar
 ---
 
-<TopBlock>
+## Overview
 
-Platform Console enables you to integrate the latest Prisma Data Platform products into your application
+Platform Console enables you to integrate the latest Prisma Data Platform (PDP) products into your application
 
 - [Accelerate](/accelerate/what-is-accelerate): Global database cache with scalable connection pooling
 - [Pulse](/pulse/what-is-pulse): A managed Change Data Capture (CDC) service that enables real-time database subscriptions
 
-[Go to Platform Console](https://console.prisma.io)
+Open the [Platform Console](https://console.prisma.io).
 
-</TopBlock>
+## Concepts
+
+The Platform Console workflows are based on four main concepts:
+
+- [**Account**](#account): In order to use PDP products, you need to have a PDP account. An _organization_ will typically create one account to manage all their workspaces, projects and environments.
+- [**Workspaces**](#workspace): An account can have multiple workspaces. A workspace typically represents a _team_ within an _organization_. **Billing happens on a per-workspace basis**, i.e. an _organization_ receives an invoice at the end of the month that captures all cost associated with the projects in a given workspace.
```

Comment:
```suggestion
- [**Workspaces**](#workspace): A user can belong to multiple workspaces. A workspace typically represents a _team_ of individuals working together on one or more projects. **Billing is on a workspace level**, i.e. the invoice for a workspace at the end of the month captures all costs associated with the projects in a given workspace.
```

I tried simplifying a little. I believe account might be better named "user", and I'd avoid bringing in the term/concept "organization".